deploy/kubernetes/elastic-agent-standalone-kubernetes.yaml

```diff
@@ -683,13 +683,13 @@ spec:
       #      - -c
       #      - >-
       #        mkdir -p /etc/elastic-agent/inputs.d &&
-      #        wget -O - https://github.com/elastic/elastic-agent/archive/8.10.tar.gz | tar xz -C /etc/elastic-agent/inputs.d --strip=5 "elastic-agent-8.10/deploy/kubernetes/elastic-agent-standalone/templates.d"
+      #        wget -O - https://github.com/elastic/elastic-agent/archive/8.10.0.tar.gz | tar xz -C /etc/elastic-agent/inputs.d --strip=5 "elastic-agent-8.10.0/deploy/kubernetes/elastic-agent-standalone/templates.d"
```

Comment:
This `https://github.com/elastic/elastic-agent/archive/8.10.0.tar.gz` link gives `404`.